examples/ai-core/src/generate-text/anthropic-chatbot-with-tools.ts

```diff
@@ -49,14 +50,7 @@ async function main() {
 
     process.stdout.write('\n\n');
 
-    messages.push({
-      role: 'assistant',
-      content: [{ type: 'text', text }, ...(toolCalls ?? [])],
-    });
-
-    if (toolResults.length > 0) {
-      messages.push({ role: 'tool', content: toolResults });
-    }
```

Comment:
This is in essence the benefit of this PR. It'll make automatic tool calling loops in agents simpler.